### ISSUE TITLE
[Feature]: Option for automatic local backup

### ISSUE THREAD

**Issue by joeljacob24** [fast4x/RiMusic#4665]
### Description

Please add an option for automatic backup for favourites, playlists, and app settings. Frequency of backup can be set to daily or weekly.

### Suggested Solution

On the miscellaneous page, under backup options, add option for automatic local backup. The folder and subfolder can be chosen by the user

**Comment by fast4x** [comment]
For now you can make backup...

**Comment by joeljacob24** [comment]
Yes I know that. But what I don't know is when the app will crash and won't respond at all. The only solution being uninstalling the app and then reinstalling it thereby losing all your data. I just suggested a feature that I have seen in other YouTube clients so that when that situation comes you won't lose all your data

**Comment by OrangeZXZ** [comment]
The open-source "Authenticator" app has a similar feature.

It creates automatically save files with different names in a folder of choice every week or more/less.

**Comment by siggi1984** [comment]
> The open-source "Authenticator" app has a similar feature.
> 
> It creates automatically save files with different names in a folder of choice every week or more/less.

sounds good, can you share a link to the app

**Comment by OrangeZXZ** [comment]
> > The open-source "Authenticator" app has a similar feature.
> > 
> > It creates automatically save files with different names in a folder of choice every week or more/less.
> 
> sounds good, can you share a link to the app

Looks like now it is called "Stratum" :
https://apt.izzysoft.de/fdroid/index/apk/com.stratumauth.app
